s3/error/s3errors.go

```diff
@@ -694,23 +700,23 @@ var ErrorCodeResponse = map[S3ErrorCode]S3ErrorStruct{
 		HttpStatusCode: http.StatusInternalServerError,
 	},
 	ErrGetBackendFailed: {
-		AwsErrorCode: "GetBackendFailed",
-		Description: "Backend is not exist, or get it failed.",
+		AwsErrorCode:   "GetBackendFailed",
+		Description:    "Backend is not exist, or get it failed.",
 		HttpStatusCode: http.StatusInternalServerError,
 	},
 	ErrUnmarshalFailed: {
-		AwsErrorCode: "UnmarshalFailed",
-		Description: "Unmarshal failed.",
+		AwsErrorCode:   "UnmarshalFailed",
+		Description:    "Unmarshal failed.",
 		HttpStatusCode: http.StatusInternalServerError,
 	},
 	ErrGetBucketFailed: {
-		AwsErrorCode: "GetBucketFailed",
-		Description: "Bucket is not exist, or get it failed.",
+		AwsErrorCode:   "GetBucketFailed",
+		Description:    "Bucket is not exist, or get it failed.",
 		HttpStatusCode: http.StatusInternalServerError,
 	},
 	ErrDBError: {
-		AwsErrorCode: "InternalError",
```

Comment:
here, can we return the actual error on the DB, like invalid SQL etc...Just thinking aloud, what would a recipient do with "InternalError" or "DBError", how to locate the root cause of the error ?

Reply:
Let's think this comment and the last one from you together, IMHO, the best way to handle error should be like this: